client/assets/default_data/panels/Panel Attack Sharp/config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id":"panel-attack-sharp",

```

Comment:
I think making the ids of the new sets more verbose would be good, if only to set an example and follow the recommendation in our docs? Just putting your nickname in the id should be enough.